### DISCUSSION TITLE
MCMA Conference 2022

### DISCUSSION THREAD

**Discussion by ftessier** [nrc-cnrc/EGSnrc#761] [Announcements]
We are happy to relay the announcement for the **International Conference on Monte Carlo Techniques for Medical Applications** (MCMA) which will be held **April 11-13 2022** at the **University of Antwerp** in the centre of the great historical port city of Antwerp, Belgium. This event was originally planned (but not announced publicly) for spring 2021 but was postponed due to covid-19.

- Please consult the conference website, coming soon: https://mcma2022.com/
- The deadline for abstract submission is December 1 2021
- ~~The early registration deadline is January 14 2022~~ Extended early registration deadline: **February 28, 2022.**
- Late registration deadline:  **March 1, 2022**
- Participation is limited to 250 participants. Covid measures may even reduce this number further
- This conference is not a hybrid event: there will be online streaming, but no opportunity to present or interact remotely

**Comment by ojalaj** [comment]
@ftessier Any news on next MCMA?!

**Comment by ftessier** [comment]
@ojalaj, we are looking forward to organizing it in Ottawa in 2025, probably late September or early October.

**Reply by ojalaj** [reply]
@ftessier Great! Just checking whether I should already start booking flights, but maybe I'll wait for another year or so
